src/CommonITILValidation.php

```diff
@@ -453,6 +453,7 @@ public function post_updateItem($history = true)
                     'id'                    => $this->fields[static::$items_id],
                     'global_validation'     => self::computeValidationStatus($item),
                     '_from_itilvalidation'  => true,
+                    'itemtype'              => static::$itemtype
                 ];
```

Comment:
```suggestion
                    'id'                    => $this->fields[static::$items_id],
                    'global_validation'     => self::computeValidationStatus($item),
                    '_from_itilvalidation'  => true,
                ] + $item->fields;
```

Reply:
I wonder if we shouldn't simply pass the `fields` (from related item) in  `$input` 

@cedric-anne  what do you think?

Reply:
`$item` is a Ticket or a Change. If we update it with all its fields, isn't there a risk of triggering other rules?

Reply:
yes, that's why I want cedric's opinion, because it may give rise to a PR on `main` (and therefore `feature request`).